UI/window-basic-main-outputs.cpp

```diff
@@ -1073,6 +1075,11 @@ bool SimpleOutput::StartReplayBuffer()
 	return true;
 }
 
+void SimpleOutput::SaveReplayBuffer()
+{
+	// do nothing
```

Comment:
To my knowledge, this issue also occurs in Simple Output Mode.  Why is this not addressed here?